examples/reference-entity-model/gradle-dhs.properties

```diff
@@ -0,0 +1,19 @@
+mlHost=
```

Comment:
I don't think it's worth checking this in, because everyone will either need to modify mlHost in this file or remember to add -PmlHost= to the command line. I think for now, it's better for this file to be gitignored. 

What we may want to add as a testing convenience is an e.g. "hubUseDhsDefaults=true" property so that people don't need to define the dozen or so properties below. That would make life easier in a number of ways.

Reply:
We would use this project to spin up a DHS instance and make this project available in HC running against a DHS. Hence the dhs environment property file.